config/initializers/cookie_rotator.rb

```diff
@@ -0,0 +1,20 @@
+# License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
+Rails.application.config.after_initialize do
```

Comment:
Needed for changing the key generator in rails 7 - https://guides.rubyonrails.org/v7.0/upgrading_ruby_on_rails.html#key-generator-digest-class-changing-to-use-sha256